types/d3-scale-chromatic/index.d.ts

```diff
@@ -1,6 +1,9 @@
 // Type definitions for D3JS d3-scale-chromatic module 1.0
```

Comment:
Since d3/d3-scale-chromatic@ade54c1 is part of v1.1.1, let's update this header to say 1.1 instead of 1.0.